### PR TITLE
Mark OPMEXTRA as not required restart data.

### PR DESCRIPTION
**Pull request by joakim-hove**
Followup to: https://github.com/OPM/opm-output/pull/174

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this pom-output=174 please

**Comment by joakim-hove** [comment]
jenkins build this opm-output=174 please

**Comment by joakim-hove** [comment]
I really do not understand what is wrong with this minor patch - is master broken?

File: opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp [hunk 1]
```diff
@@ -427,7 +427,7 @@ namespace Opm
                                                                   {"TEMP"     , UnitSystem::measure::temperature},
                                                                   {"RS"       , UnitSystem::measure::gas_oil_ratio},
                                                                   {"RV"       , UnitSystem::measure::oil_gas_ratio}};
-        std::set<std::string> extra_keys {"OPMEXTRA"};
+        std::map<std::string, bool> extra_keys {{"OPMEXTRA"} , false};
```
Comment (bska) [inline opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp]: Don't you need to say
```C++
std::map<std::string, bool> extra_keys {{"OPMEXTRA", false}};
```
instead?

Comment (joakim-hove) [inline opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp]: Ehhh - yes!

**Comment by joakim-hove** [comment]
jenkins build this opm-output=174 please

**Comment by joakim-hove** [comment]
jenkins build this opm-output=174 please

**Comment by joakim-hove** [comment]
@atgeirr : Will you merge this - and the upstream changes?

**Comment by atgeirr** [comment]
> Will you merge this - and the upstream changes?

Will try to get to it, if not now then tonight.

**Comment by atgeirr** [comment]
Looks fine, merging! @dr-robertk there is still some changes on the opm-simulator side (to handle the case of missing OPMEXTRA) required before you can run with your existing restarts.

**Pull request merged**